en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/oauth2-access-tokens.md

```diff
@@ -1,15 +1,25 @@
-# Secure APIs using JWT - Self Contained JWT
+# OAuth 2.0 Authentication
 
-Choreo Connect can accept JWTs issued by a **trusted** key manager as a valid token to invoke the APIs.  JWT tokens are self-validated by Choreo Connect without validating it against the authorization server(key manager) that issued the JWT.
+By default all APIs expect an access token during invocation, unless [security has been disabled]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/disabling-security/) for the API.
```

Comment:
Shall we add something about OAuth in this introduction? This is too disconnected from the title for a reader.

Reply:
Yeah, it is.. I'll try to make it sound better.